tutorials/deploy-local/inferenceconfig.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Can you make a second inferenceconfig.json that uses the non-dummy entry script? Follow this example: https://docs.microsoft.com/en-us/azure/machine-learning/how-to-deploy-and-where?tabs=azcli#define-an-entry-script.

The idea is that the CLI deployment should follow the exact same steps as the Python one. Currently you're missing the step where we deploy with the real model (you're still using the echo model).

Reply:
- included the non-dummy entry script with name inferenceconfig.json
- included the dummy entry script config JSON as dummyinferenceconfig.json